.travis.yml

```diff
@@ -60,18 +62,19 @@ jobs:
     - stage: lint
       name: Lint (PHP, JavaScript, and configuration files)
       php: "7.4"
-      env: WP_VERSION=latest DEV_LIB_ONLY=xmllint,phpsyntax
+      env: WP_VERSION=latest DEV_LIB_ONLY=xmllint,phpsyntax,composer
       script:
         - source "$DEV_LIB_PATH/travis.script.sh"
         - npm run lint
 
     - stage: test
       name: E2E tests with Docker (7.4, WordPress latest, with code coverage)
       php: "7.4"
-      env: NODE_ENV=e2e WP_VERSION=latest
```

Comment:
`WP_VERSION` is used both by our `docker-compose.yml` setup and the wp-dev-lib library. Unfortunately, the Docker reference container doesn't have an image for the `latest` tag, so we rely on the `WP_VERSION` version set in `.env`.